components/cf-app-push/frontend/src/view/deploy-app-workflow/deploy-app.scss

```diff
@@ -0,0 +1,86 @@
+.modal.detail-view.detail-view-dialog>.modal-dialog>.modal-content .detail-view-content.deploy-app wizard .wizard-step {
+  padding-bottom: 24px;
+  padding-left: 24px;
+}
+.deploy-app {
+  deploy-location {
+    .control-title {
+      margin-top: 0;
+    }
+  }
+
+  #deploy-app-choose-bits {
+    display: none;
+  }
+
+  .deploy-app-github {
+    display: flex;
+
+    .deploy-app-github-form {
+      .deploy-app-github-branch {
+        margin-bottom: 100px;
```

Comment:
In conjunction with the max-height of the dropdown-menu. It means the dialog allows enough space at the bottom to show the drop down when open, otherwise scrolling is required. The actual figure seemed about the right number of lines